src/components/TaskContainer.jsx

```diff
@@ -1,22 +1,35 @@
 import AddTask from 'components/AddTask';
 import NoTaskPlaceholder from 'components/NoTaskPlaceholder';
 import Task from 'components/Task';
+import Button from 'components/ui/Button';
+import { useState } from 'react';
 import { useSelector } from 'react-redux';
+import { paginationLabel } from 'utils/helpers';
+import { nextPage, paginate } from 'utils/helpers/ReducerHelper';
```

Comment:
We should move the react based imports to the top of the file.

Reply:
Import re-ordered